openff/toolkit/utils/builtin_wrapper.py

```diff
@@ -131,7 +131,7 @@ def assign_partial_charges(
             partial_charges = [0.0] * molecule.n_atoms
 
         elif partial_charge_method == "formal_charge":
-            partial_charges = [float(atom.formal_charge.m) for atom in molecule.atoms]
+            partial_charges = [float(atom.formal_charge.m) for atom in molecule.atoms]  # type: ignore
```

Comment:
What's the error code here?

I won't be picky enough to ask here, but in a perfect world I'd love to see us always have the specific error code in these ignores. [Mypy can enforce this](https://mypy.readthedocs.io/en/stable/error_codes.html#requiring-error-codes)